### DISCUSSION TITLE
Heat of combustion Total or Chemical?

### DISCUSSION THREAD

**Discussion by Myrmecio** [firemodels/fds#13666] [Q&A] [answered]
Hi everyone,

I'm working on an FDS simulation focused on the evolution of the smoke and hot gas layer. I'm using a burner with a prescribed heat release rate following a t-squared curve.

I have a question regarding the HEAT_OF_COMBUSTION parameter. Should I use the total heat of combustion or the chemical heat of combustion, according to tables A.38 and A.39 of the SFPE handbook of fire?

Also, should I calculate the radiative fraction by dividing the chemical heat of combustion by the radiative heat of combustion, or should I use the default FDS value of 0.35? I noticed that the predefined reactions in the Pyrosim library have specific values for these parameters, which has raised this question.

Thanks in advance for your insights!

**Comment by mcgratta** [comment]
Use what is know as the "lower" heating value. In the latest handbook, it is the "net" Deltah_c.

As for radiative fraction, there are a few suggested values for a few fuels. Use those or choose your own. Typical fires are about 35%.

**Comment by Myrmecio** [comment]
Thank you for your response, Kevin.

My question arises because for some versions of Pyrosim, some reactions from the SFPE Handbook, 5th Edition, Tables A.38 and A.39, have been loaded into the reaction library, where it can be seen how the heat of combustion is taken as that shown in the ΔHch column instead of the ΔHcT. Additionally, the radiative fraction is specified as the ratio between ΔHc_rad/AHch from Table A.39 of the SFPE Handbook, 5th Edition (as well as different yields to model combustion such as soot yield and CO yield).

Is the information provided correct?
On the other hand, many fuels appear in this table which are quite common in design fires. Is it appropriate to use the values of ΔHch and ΔHc_rad/AHch to model the radiative fraction in the combustion reaction?


![image](https://github.com/user-attachments/assets/03b02e06-5e00-434d-a44d-5c411f2247fc)

Note: These are models whose purpose is to verify the movement of smoke and hot gases in large volumes such as logistics storage buildings in order to verify the buoyancy of the hot layer, whether or not destratification occurs, and to calculate the maximum size of smoke reservoirs by means of hanging curtains, which is the usual practice in Spain, as well as the ganged activation of roof vents in the building to confine the smoke in the smoke reservoir where the fire occurs and to study whether the smoke spills over into adjacent reservoirs.

**Comment by drjfloyd** [comment, marked as answer]
The Delta H_T column in Table A.38 is for complete combustion, i.e., no minor products.  This would be the IDEAL=T value in FDS.  It has never been very clear to me what the Delta H_ch is in Table A.39.  For example, based on heats of formation propane with 0.5 % CO yield and 2.4 % soot yield should have a heat of combustion of ~45.5 MJ/kg and Table A.39 has 43.7 MJ/kg.
